modules/@apostrophecms/asset/index.js

```diff
@@ -56,65 +142,56 @@ module.exports = {
   tasks(self) {
     return {
       build: {
-        usage: 'Build Apostrophe frontend javascript master import files',
+        usage: 'Build Apostrophe frontend CSS and JS bundles',
         afterModuleInit: true,
         async task(argv) {
           const namespace = self.getNamespace();
           const buildDir = `${self.apos.rootDir}/apos-build/${namespace}`;
-          const modulesDir = `${buildDir}/modules`;
           const bundleDir = `${self.apos.rootDir}/public/apos-frontend/${namespace}`;
           // Don't clutter up with previous builds.
           await fs.remove(buildDir);
           await fs.mkdirp(buildDir);
-          await fs.remove(modulesDir);
-          await fs.mkdirp(modulesDir);
 
-          let rebuildAposUi = argv && !argv['check-ui-build'];
+          for (const [ name, options ] of Object.entries(self.options.builds)) {
+            let rebuild = argv && !argv['check-ui-build'];
 
-          const APOS_MERGED_BUNDLE = 'apos-bundle.js';
-          const APOS_ONLY_BUNDLE = 'apos-only-bundle.js';
-          const APOS_ONLY_TS = '.apos-only-timestamp.txt';
-          const PUBLIC_BUNDLE_CSS = 'public-bundle.css';
-          const PUBLIC_BUNDLE_JS = 'public-bundle.js';
-          let checkTimestamp = false;
+            let checkTimestamp = false;
 
-          const bundleExists = await fs.pathExists(bundleDir);
+            if (options.core) {
```

Comment:
Only the apos build currently has the core flag, others will rebuild all the time, like ui/public always did. The expectation is that they're small.